test/Unit/Porter/Net/Http/HttpOptionsTest.php

```diff
@@ -60,6 +60,11 @@ public function testReplaceHeaders()
         self::assertCount(2, $options->getHeaders());
     }
 
+    public function testProxy()
+    {
+        self::assertSame('https://foo.com:80', (new HttpOptions)->setProxy('https://foo.com:80')->getProxy());
```

Comment:
One thing I encourage developers to do is avoid specifying constants more than once in a test case. This slightly reduces maintenance cost if, for example, strings need to be updated in future in order for tests to pass. For example,

```php
self::assertSame($host = 'https://foo.com:80', (new HttpOptions)->setProxy($host)->getProxy()); 
```

You can see this pattern used throughout the other test cases in this file. Also applies to `SoapOptionsTest`.